tests/test_base_interface.py

```diff
@@ -21,7 +21,7 @@ def pfbi(pf_app):
 
 @pytest.fixture
 def activate_test_project(pfbi):
-    pfbi.app.ActivateProject(r"\seberlein\powfacpy\powfacpy_tests")
+    pfbi.app.ActivateProject(r"\mfranke\powfacpy\powfacpy_tests")
```

Comment:
Please dont submit this in future. I have a solution to this problem using GetCurrentUser().